server/testserver.go

```diff
@@ -121,12 +118,12 @@ func (ts *TestServer) Start() error {
 
 // Stop stops the TestServer.
 func (ts *TestServer) Stop() {
-	ts.stop()
+	ts.Server.Stop()
 }
 
-// Sets the retry options for stores in TestServer.
+// SetRangeRetryOptions sets the retry options for stores in TestServer.
 func (ts *TestServer) SetRangeRetryOptions(ro util.RetryOptions) {
-	ts.server.node.lSender.VisitStores(func(s *storage.Store) error {
+	ts.Server.node.lSender.VisitStores(func(s *storage.Store) error {
```

Comment:
You don't strictly need to specify ts.Server here, since the *Server is embedded in TestServer.